Assets/HoloToolkit/UX/Prefabs/Text.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 59c43794cfe522246aaa1d4a04cadf68
```

Comment:
What's this from?

Reply:
I missed adding this file as part of #2052